code/modules/power/singularity/singularity.dm

```diff
@@ -12,49 +12,50 @@
 
 
 /obj/machinery/singularity/
-	name = "Gravitational Singularity"
-	desc = "A Gravitational Singularity."
+	name = "gravitational singularity" //Lower case

```

Comment:
Upper case causes problems with \the macros. I'd rather have it lower cased, unless we are going to put it in upper case but slap it with /improper